set.mm

```diff
@@ -469434,6 +469516,82 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+      MUWEAUWAUWMVQZUWAUWEAVRLZOZUWEUWIUHZUWEABUWEPZVSZUWEUWIAUWIPZVTQZWAWBWCWG
+      WDZWEUWTUYOUYCIZUXCUYHUWAUWOVUKUWCUWAUWOJUWMUWEFAUWFUWNUWLUYAUWMPUWAAWHOZ
+      UWOUWAAWIOZVULABWJZVUMVULAWKOZAWLZWPZQMUWAVUDUWOVUGMUWFPZUWAUWFWHOZUWOUWA
+      BUWFWHUWHUWABWIOZBWHOZABWMZVUTVVABWKOBWLWPZQWNMUWAUWOTWOWQZSUYNUYSUYKUXAU
+      YPGZUYMUYNUYQUWIVVEFAUXAWRVUHUWAAWSOZUWCUWOUXCUYHUWAAWTOZVVFUWAVUMVULVVGV
+      UNVUQAXAXBZAXCQXDUXAWROUYNDXEZXFUYNVVGUYKUWIOZUXAUWIOZVVEUWIOUXIVVGUYHUWA
+      VVGUWCUWOUXCVVHXHZMZUYNUWEUWIUYKUXIVUEUYHUWAVUEUWCUWOUXCVUIXHZMUYNUYKUYFU
+      WEUYNUWLUYFUXAUXNUYHUWLUYFUXNXIUXIUXNUYFUWLXGXJUYNUXAUXBUWLDXKUYTXLZXMUWA
+      UWEUYFIUWCUWOUXCUYHUWABUXOKUWABUWFUXOUWHUWAUWMUWEAVUBVUIXNXOXPXDXQZXRZUXI
+      VVKUYHUXIUXAUWMKLZUWIUXIUXBVVRUHUXAVVROUXIUXBUWLVVRUWTUXCTUXIUWOUWLVVRUHU
+      WDUWOUXCVPZUWLUWNVVRUWMVVRPZUWSXSQXTUXAVVRVVIYAYBUXIUWMUWEAUXIUWMVQVVNYCZ
+      XQMZUWIAUYPUYKUXAVUHUYPPZYDYEUYNUXRUXAIZJZUXSUYKUXRUXAUYPVWEUXRUXAUXNUYNV
+      WDTZXPVWFYFYHZUYNUYLUYPUYKUXAUWAUYLUYPIUWCUWOUXCUYHUWAUYLUWFVOLZUYPUWABUW
+      FVOUWHXPUWAVUDUYPVWHIVUGUWEAUWFUYPVUCVURVWCYGQYIXDWGYIYJWQUYIBWTOZUYKUWEO
+      ZUXAUWEOZUYMUWEOUWAVWIUWCUWOUXCUXKUYHUWAVUTVVAVWIVVBVVCBXAXBWCUXIUYHVWJUX
+      KVVPWQUXIVWKUXKUYHUXIUXQAYKLZUYCIZJZVWKEUYGUYNVWMVWKUXQUYNVWMJZUXAAYLLZLZ
+      VWPLZUXAUWEUYNVVGVWMUXAAYMLZOZVWRUXAIVVMVWOVWTVWQUYKIZVWOUWIAUYPVWSVWLVWP
+      UXAUYKVUHVWCVWLPZVWSPZVWPPZVWOUWAVVGUWAUWCUWOUXCUYHVWMUUAZVVHQUYNVVKVWMVW
+      BMZUYNVVJVWMVVQMZVWOUXAUYKUYPGZVVEVWLVWOVUOVVKVVJVXHVVEIVWOUWAVUMVUOVXEVU
+      NVUMVULVUOVUPUUBXBVXFVXGUWIAUYPUXAUYKVUHVWCUUCYEVWOVWLUYSVVEVWOVWLUYCUYOU
+      YSUYNVWMTUWTVUKUXCUYHVWMVVDXHVWOUYBUYRARUYNUYBUYRIVWMVUJMWEUUDUYNUYSVVEIV
+      WMVWGMXOZYIVWOVWLVVEVXIWBUUIZUUEAVWSVWPUXAVXCVXDUUFYNVWOVULVUDVUSVWQUWEAV
+      JLZURUUGZOZVWRUWEOZVWOUWAVUMVULVXEVUNVUQXBZVWOUWAVUDVXEVUGQVWOBUWFWHVWOUW
+      AUWGVXEUWHQVWOUWAVUTVVAVXEVVBVVCXBWNVWOVWQUWEOVWQVXKUKZVXMVWOVWQUYKUWEVWO
+      VWTVXAVXJUUHUYNVWJVWMVVPMYOVWOVULVVKUXAVXKUKZVXPVXOVXFUYNDCWFVWMVXKUWLOYS
+      ZVXQVVOUWTVXRUXCUYHVWMUWTVXKUWMVJLZUWLUWAVXKVXSIUWCUWOUWAUWMUWEAVXKVUBUWA
+      VXKVQVUIUUJSUWDUWQUWOUWLUWMUUKLZOVXSUWLOYSUWRUWTUWNVXTUWLUWNUWMUWSUULUXGU
+      UMUWLUWMVXSVXSPUVGYNUUNXHUXAVXKUWLUUOYNUWIAVWPUXAVXKVUHVXKPZVXDUURYEVWQUW
+      EVXKUUPUUQVULVUDJZVUSJZVXMJUBNZVWPLZUWEOZVXNUBVXLVWQVYDVWQIVYEVWRUWEVYDVW
+      QVWPUUSUUTVYCVYFUBVXLUVAZVXMVYBVUSVYGUBUWEAUWFVWPVXKVURVYAVXDUVBVGMVYCVXM
+      TUVCUVDWNUVEUXIVWLVVROVWNEUYGYPUXIVWLUWIVVRUXIVVGVWLUWIOVVLUWIAVWLVUHVXBU
+      VFQVWAUVHUXIFVVRUXOUXTUYFUWMUWLVWLUXPEVVTUYFPZUXOPZUXPPZUXTPZUXIUWQUWMUVI
+      OUWDUWQUWOUXCUWRSUWMUVJQZVVSYQUVKYRSUWEBUYLUYKUXAVUFUYLPYDYEYOSYOUVLUXIUX
+      KUYEEUYGYPZUXIUXKUXJVVROVYMUXIUWIVVRUXJVWAUVMUXIFVVRUXOUXTUYFUWMUWLUXJUXP
+      EVVTVYHVYIVYJVYKVYLVVSYQUVNVGYRUVOUVPYTYTUWDUWJJZUWJVUMUWEWROUWKUWDUWJTUW
+      AVUMUWCUWJVUNSVYNBKUVQUWEUWIUWFAWIWRVURVUHUVRYEUVSUVT $.
+  $}
+
+  $( The degree of the extension ` E /FldExt F ` is ` 1 ` exactly then ` E `
```

Comment:
```suggestion
  $( The degree of the extension ` E /FldExt F ` is ` 1 ` exactly when ` E `
```

(or some other wording like "iff")

Reply:
I would prefer "iff" instead of then/when

Reply:
I'll use "iff" since both of you suggest it.